### PR TITLE
Privacy banner update

### LINKED ISSUE #3446: Change the privacy banner's cookie logic
**Is your feature request related to a problem? Please describe.**
The logic for the privacy banner needs to change - instead of tracking whether the user has seen it, we want to track which version of the privacy policy was in place when the banner was seen.

**Describe the solution you'd like**
Change the privacy banner logic so that:
- if a user has no ckns_privacy cookie, display the banner
- if a user has the ckns_privacy cookie, but its value is `0` or `1` then display the banner
- if a user has the ckns_privacy cookie, and its value is `july2019` then do not display the banner
- if a user has the ckns_privacy cookie, and its value is not `0`, `1`, or `july2019` then do not display thhe banner (as the assumption here is that it has a future value we don't know about yet)
- when the banner is displayed to the user, set the ckns_privacy cookie value to `july2019`

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by sareh**
Resolves #3446

**Overall change:** Updates Privacy banner logic according to these rules:

Cookie name | Value | Behaviour of Privacy banner
-- | -- | --
ckns_privacy | not present or value is undefined or null | Show banner
ckns_privacy | '0' or '1' (e.g. previousVersion) | Show banner
ckns_privacy | 'july2019' (e.g. currentVersion) | Do not show banner
ckns_privacy | anything else | Do not show banner

**Code changes:**

- Update logic & add unit tests. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/ConsentBanner/CanonicalLogic/index.js [hunk 1]
```diff
@@ -19,15 +21,22 @@ const setPolicyCookie = (value, logger) => {
...
 
-const seenPrivacyBanner = () => Cookie.get(PRIVACY_COOKIE) === BANNER_APPROVED;
-const seenCookieBanner = () => Cookie.get(EXPLICIT_COOKIE) === BANNER_APPROVED;
+const showPrivacyBanner = () => {
+  const privacyCookie = Cookie.get(PRIVACY_COOKIE);
+  return (
+    !privacyCookie || PRIVACY_COOKIE_PREVIOUS_VALUES.includes(privacyCookie)
+  );
+};
+const seenCookieBanner = () =>
```
Comment (dr3) [inline src/app/containers/ConsentBanner/CanonicalLogic/index.js]: Can this be renamed to be `showCookieBanner` for consistency please?

**Review by dr3 — CHANGES_REQUESTED** [review]
https://github.com/bbc/simorgh/blob/latest/cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js needs updating too to match new cookie setting

**Review by dr3 — APPROVED** [review]
LGTM assuming cypress and other CI passes

**Review by jamesbhobbs — APPROVED** [review]

**Comment by sareh** [comment]
Tested locally:
1) On first visit, shows privacy banner and adds cookie with value `july2019`

Screenshot 2019-08-29 at 12 29 25
2)
Screenshot 2019-08-29 at 12 29 47

**Pull request merged**